### ISSUE TITLE
Old repo but anyways

### ISSUE THREAD

**Issue by addelovein** [makton-dev/Cloudbase-PVE-Mods#1]
CD-ROM Solution

---- Powershell ----
$drive_letter="F"
$vol = get-wmiobject -Class Win32_Volume | where{$_.Name -eq $drive_letter+":\"}  
$vol.DriveLetter = $null
$vol.Put()  
$vol.Dismount($false, $false)

**Comment by makton-dev** [comment]
@addelovein It's old, but I still use it for my 6 Node PVE cluster.

Thank you for the info above. Take it you tested this on your environment already?

**Comment by addelovein** [comment]
> @addelovein It's old, but I still use it for my 6 Node PVE cluster.
> 
> Thank you for the info above. Take it you tested this on your environment already?

Yes I tested it. However I went another route in the end and created a cloud-init client in pure Powershell and only for proxmox.

**Comment by makton-dev** [comment]
Thank you for the solution. I will add it to this repo. Am moving away from my On-Prem Prox environment and going Raspberry PI K3S 8 node Cluster and using Cloud (AWS and Azure). I still have other Environments that use Proxmox and Windows, making this still viable.
